src/CreditCardForm.php

```diff
@@ -32,6 +32,7 @@ public function form(array $form, array &$form_state, \Payment $payment) {
       'client_secret' => $intent->client_secret,
       'intent_type' => $intent->object,
       'pmid' => $method->pmid,
+      'css_src' => variable_get('stripe_payment_css_src', []),
```

Comment:
I'd strongly suggest to put "font" in the variable name, e.g. "stripe_payment_font_src", since it’s only meant to be used for font related CSS.